src/messaging/tests/TestSecureChannelMgr.cpp

```diff
@@ -0,0 +1,272 @@
+/*
+ *
+ *    Copyright (c) 2020-2021 Project CHIP Authors
```

Comment:
If keeping this should be just 2021 probably.

However I would still remove both Copyright and All rights reserved lines.